src/Symfony/Component/HttpKernel/DependencyInjection/RegisterControllerArgumentLocatorsPass.php

```diff
@@ -65,6 +65,14 @@ public function process(ContainerBuilder $container)
             $class = $parameterBag->resolveValue($class);
 
             if (!$r = $container->getReflectionClass($class)) {
+                if ($id === $class) {
+                    /*
+                     * This happens when container has not been built before and if $class is missing some use-statements.
+                     * Make sure PHP loads the file to show any exceptions the user.
```

Comment:
`[...] exceptions the user` -> `[...] exceptions to the user`

Reply:
Thank you